tests/integration/test_powershell_tool/test_terminal_timeout_advanced.py

```
@@ -96,24 +98,24 @@ def main():
                         first_bytes = f.read(20)
                     print(f"First bytes: {first_bytes}")
                 else:
-                    print("❌ create_sample.py was NOT created")
+                    print("âŒ create_sample.py was NOT created")
 
             except TimeoutError:
                 elapsed = time.time() - start_time
-                print(f"❌ Command timed out after {elapsed:.2f}s")
+                print(f"âŒ Command timed out after {elapsed:.2f}s")
                 print("Confirming the timeout issue exists")
             except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)